_docker/drupal/drupal-filesystem/composer.json

```diff
@@ -101,7 +100,7 @@
         "drupal/akamai": "3.0-alpha4",
         "drupal/trailing_slash": "^1.0",
         "drupal/bootstrap": "3.17",
-        "drupal/compose": "1.x",
+        "drupal/compose": "1.x-dev",
```

Comment:
@jordanpagewhite Same comment as above, can we please lock the dependency to a known good commit in the source repo.

cheers,

Rob

Reply:
@robpblake I resolved this in the commit I just pushed. Thanks!